pipelines/skylab/multiome/Multiome.changelog.md

```diff
@@ -1,5 +1,10 @@
-# 6.1.1
-025-04-15 (Date of Last Commit)
+# 6.0.2
```

Comment:
Since 6.0.1 has not been released yet, we can consolidate these to both be part of 6.0.1